package.json

```diff
@@ -49,6 +49,7 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-uglify": "^2.0.0",
     "ember-cli-update": "^0.14.0",
+    "ember-copy": "^1.0.0",
```

Comment:
@pauln can you move this to regular dependencies and update the yarn.lock file

Reply:
Done!